src/main/java/edu/iu/terracotta/repository/QuestionRepository.java

```diff
@@ -11,7 +11,7 @@
 
 public interface QuestionRepository extends JpaRepository<Question, Long> {
 
-    List<Question> findByAssessment_AssessmentId(Long assessmentId);
+    List<Question> findByAssessment_AssessmentIdOrderByQuestionId(Long assessmentId);
```

Comment:
I think this should order by `questionOrder`. Looks like this would order by `questionId`?